usage/src/com/cloud/usage/UsageSanityChecker.java

```diff
@@ -72,7 +72,9 @@ protected boolean checkItemCountByPstmt(CheckCase checkCase) throws SQLException
         try (PreparedStatement pstmt = conn.prepareStatement(checkCase.sqlTemplate)) {
             if(checkCase.checkId) {
                 pstmt.setInt(1, lastId);
-                pstmt.setInt(2, maxId);
+                if (maxId > 0) {
+                    pstmt.setInt(2, maxId);
```

Comment:
Hi @rafaelweingartner sorry I haven't seen you comment earlier, thanks for reviewing!
Actually the first parameter on line 74 belongs to the one set on line 216, and the one inside the `if` is only set if it was set on the query, on line 194.